src/main/java/com/github/wovnio/wovnjava/UrlResolver.java

```diff
@@ -7,16 +7,21 @@ private UrlResolver() {}
 
     static String computeClientRequestUrl(HttpServletRequest request, Settings settings) {
         String scheme = request.getScheme();
-        String host = clientRequestHostAndPort(request, settings.useProxy);
+        
+        if (!settings.fixedScheme.isEmpty()) {
+            scheme = settings.fixedScheme;
+        }
+
+        String host = clientRequestHostAndPort(request, settings);
```

Comment:
I think this reads better if you structure it like (loosely, this isn't valid java)
```
String host = overrideHost() || clientRequestHostAndPort(request, settings)
String scheme = settings.fixedScheme || request.getScheme()
```

That way `clientRequestHostAndPort` is really about reading the data from the actual request, and is separated from the overrides